curriculum/challenges/english/25-front-end-development/quiz-html-accessibility/66ed9026f45ce3ece4053ebb.md

```diff
@@ -17,439 +17,446 @@ Answer all of the questions below correctly to pass the quiz.
 
 ---
 
-Placeholder distractor 2
+<kbd>Enter</kbd>
 
 ---
 
-Placeholder distractor 3
+<kbd>Caps Lock</kbd>
 
 #### --answer--
 
-Placeholder answer
+<kbd>Shift</kbd> + <kbd>Tab</kbd>
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the function of the `aria-live` attribute?
 
 #### --distractors--
 
-Placeholder distractor 1
+The function of the `aria-live` attribute is to record live music performances 
 
 ---
 
-Placeholder distractor 2
+The function of the `aria-live` attribute is to make HTML elements focusable
 
 ---
 
-Placeholder distractor 3
+The function of the `aria-live` attribute is to make HTML elements smaller
 
 #### --answer--
 
-Placeholder answer
+Indicate an element's content is dynamic and will be updated, which enables assistive technology to mention those updates to the user.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the purpose of the `role` attribute?
 
 #### --distractors--
 
-Placeholder distractor 1
+To style elements with CSS animations
 
 ---
 
-Placeholder distractor 2
+To define the visual appearance of HTML elements
 
 ---
 
-Placeholder distractor 3
+To modify HTML tags
 
 #### --answer--
 
-Placeholder answer
+To specify the type and purpose of an element, allowing assistive technologies to better understand how the element should be interpreted and interacted with.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the best way to make a website's link accessible? 
 
 #### --distractors--
 
-Placeholder distractor 1
+Use phrases like "click here" to describe the link
 
 ---
 
-Placeholder distractor 2
+Add the site's URL to the link's text
 
 ---
 
-Placeholder distractor 3
+Link to two different URLs using the same words on the same page
 
 #### --answer--
 
-Placeholder answer
+Avoid linking whole paragraphs 
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+When recording audio for a video, it is best to:
 
 #### --distractors--
 
-Placeholder distractor 1
+Use the latest recording tool
 
 ---
 
-Placeholder distractor 2
+Use jargon and terms related to the topic
 
 ---
 
-Placeholder distractor 3
+Speak quickly 
 
 #### --answer--
 
-Placeholder answer
+Speak slowly and clearly to give users time to process and understand the presented information.  
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the function of the `aria-checked` attribute? 
 
 #### --distractors--
 
-Placeholder distractor 1
+The function of the `aria-checked` attribute is to show a checkmark. 
 
 ---
 
-Placeholder distractor 2
+The function of the `aria-checked` attribute is to mark HTML elements
 
 ---
 
-Placeholder distractor 3
+The function of the `aria-checked` attribute is to add a circle 
 
 #### --answer--
 
-Placeholder answer
+The function of the `aria-checked` attribute is to ensure that the element is checked (`true`), unchecked (`false)`, or if the checked status is indeterminate (`mixed`), neither checked nor unchecked
```

Comment:
```suggestion
The function of the `aria-checked` attribute is to indicate whether an element is checked (`true`), unchecked (`false)`, or in an indeterminate state (`mixed`), meaning neither checked nor unchecked
```

Nitpick. I just think "indicate" is better than "ensure", since this attribute is merely describing the current state to assistive technology.